app/services/fastboot.js

```diff
@@ -40,6 +40,8 @@ export default Ember.Service.extend({
     return this._fastbootInfo.request.host();
   }),
 
+  response: alias('_fastbootInfo.response'),
```

Comment:
It must be aliased. The status code and headers from the response need to
propagate back from the Ember App to the fastboot server.

On Thursday, April 28, 2016, Robert Jackson <notifications@github.com>
wrote:

> In app/services/fastboot.js
> <https://github.com/ember-fastboot/ember-cli-fastboot/pull/185#discussion_r61486609>
> :
>
> > @@ -40,6 +40,8 @@ export default Ember.Service.extend({
> >      return this._fastbootInfo.request.host();
> >    }),
> >
> > +  response: alias('_fastbootInfo.response'),
>
> Probably should be computed.readOnly?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ember-fastboot/ember-cli-fastboot/pull/185/files/0d80f7b6afae4a9e4f726c3abe2c7568eef70964#r61486609>
>


-- 
Arjan